modules/collector-source/pkg/collector/clustermap.go

```diff
@@ -0,0 +1,86 @@
+package collector
+
+import (
+	"fmt"
+
+	"github.com/opencost/opencost/core/pkg/clusters"
+	"github.com/opencost/opencost/core/pkg/log"
+)
+
+type collectorClusterMap struct {
```

Comment:
Aw man, I'm sorry about this! I forgot to add a generic cluster info adapter for ClusterMap to core. Glad you figured it out.